### ISSUE TITLE
Backgrid.Column Doesn't Support Defaults

### ISSUE THREAD

**Issue by machineghost** [wyuenho/backgrid#353]
Backgrid.Column has this line:

```
Backgrid.requireOptions(attrs, ["cell", "name"]);
```

to ensure that a cell gets provided.  That works great normally, but Backbone also allows for attributes to be provided via a "defaults" property, instead of being provided directly, and in that case the `requireOptions` fails.

In other words, while the following two code bits are equivalent:

```
var MyModel = Backbone.Model.extend();
new MyModel({foo: 'bar'});

var MyModel = Backbone.Model.extend({defaults: {foo: 'bar'});
new MyModel();
```

the same is not true if you replace `Backbone.Model` with `Backgrid.Column` (and "foo" with "cell").

This could be fixed with the following one-line change:

```
Backgrid.requireOptions(this.attributes, ["cell", "name"]);
```

In other words, if Backgrid checked `this.attributes` (which, by that point in the code has already had `defaults` applied to it) instead of `attrs`, columns would be able to have default cells/names.
